src/main/java/hannah/task/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package hannah.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    String name;
+    LocalDate deadline;
+    public Deadline(String name){
+        super(name);
+    }
+    @Override
+    public String toString(){
+        return("[D]" + super.toString() + " by " +  this.deadline);
+    }
```

Comment:
Not a coding standard issue, perhaps you will want to separate line 9 to 10 and line 12 to 13.